rpcserver.go

```diff
@@ -5825,7 +5826,18 @@ func (r *rpcServer) AddInvoice(ctx context.Context,
 				blindingRestrictions,
 			)
 		},
-		MinNumHops: r.server.cfg.Routing.BlindedPaths.NumHops,
+		MinNumBlindedPathHops: r.server.cfg.Routing.BlindedPaths.
+			NumHops,
+		DefaultDummyHopPolicy: &blindedpath.BlindedHopPolicy{
+			CLTVExpiryDelta: uint16(defaultDelta),
+			FeeRate:         uint32(r.server.cfg.Bitcoin.FeeRate),
+			BaseFee:         r.server.cfg.Bitcoin.BaseFee,
+			MinHTLCMsat:     r.server.cfg.Bitcoin.MinHTLCIn,
+
+			// MaxHTLCMsat will be calculated on the fly by using
```

Comment:
hmm, I think we can be more explicit in the comment and mention also that this only works when we are the introduction point of the route ?